web/themes/custom/pineapple/sass/layouts/nodes/_recipe-page.scss

```diff
@@ -0,0 +1,84 @@
+.row {
+  display: flex;
+  flex-wrap: wrap;
+
+  .node__content {
```

Comment:
Remove this code from the row nesting, the row shouldn't be used as a target for other selectors, you may wanna change this selector to something else

Reply:
done